### ISSUE TITLE
can the PCB board work?

### ISSUE THREAD

**Issue by chinahuangyong** [a-little-wifi/mxm-nvme#1]

**Comment by a-little-wifi** [comment]
No, this board is mostly intended to just be a reference for making your own MXM card PCB, the m.2 slots on there are just a silly idea that is nowhere near completely implemented and will not work in its current state.
